.travis.yml

```diff
@@ -1,16 +1,20 @@
 language: ruby
+perl:
```

Comment:
this is ignored for ruby projects. So if you really want perl 5.18, you need to all perlbrew yourself - the default is 5.14